File: BackendPhotoItemsApp/Models/Db/ItemPhotoPropertySet.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace BackendPhotoItemsApp.Models.Db
{
    public partial class ItemPhotoPropertySet
    {
        public int Id { get; set; }
        public int ItemPhotoId { get; set; }
        public int PropertyId { get; set; }
        public string Value { get; set; }

        public virtual ItemPhoto ItemPhoto { get; set; }
        public virtual Property Property { get; set; }
    }
}

File: frontend-photo-items-app/src/models/ItemPhotoPropertySet.ts
import ItemPhoto from "./ItemPhoto";
import Property from "./Property";

export default class ItemPhotoPropertySet {

  public id: number = 0;
  public itemPhotoId: number = 0;
  public propertyId: number = 0;
  public value: string = "";

  //Navigations
  public itemPhotoNavigation: ItemPhoto = new ItemPhoto();
  public propertyNavigation: Property = new Property();

  constructor(itemPhotoPropertySet?: Partial<ItemPhotoPropertySet>) {
    Object.assign(this, itemPhotoPropertySet);
  }

}